blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -377,7 +377,7 @@ const Nav = (props) => {
       </div>
     );
   };
-
+  const rightNav = useMemo(() => <NavSection side="right" />, []);
```

Comment:
we shouldn't need to do this any more if we're working off the most recent version of canary